tests/ping.go

```diff
@@ -101,3 +102,28 @@ func vmiConsoleExpectBatch(vmi *v1.VirtualMachineInstance, expected []expect.Bat
 	}
 	return err
 }
+
+// PingAppFromVirtHandler performs a natcat tcp check through the provided virt-handler pod.
+// Optional port and arguments for the netcat command may be provided.
+// If port is not provided, port "8080" is used.
+func PingAppFromVirtHandler(pod *k8sv1.Pod, ipAddr, port string, args ...string) error {
```

Comment:
The "ping" specifies here that it checks connectivity.

It is named `PingApp` and the "tcp" is an implementation detail here (it could be UDP with the correct arguments).
I prefer to leave it abstracted and generic. If we find this colliding with other things, we can change it.

Reply:
Ack